calibrators/intrinsic_camera_calibrator/intrinsic_camera_calibrator/intrinsic_camera_calibrator/calibrators/ceres_calibrator.py

```diff
@@ -18,19 +18,52 @@
 import threading
 from typing import Dict
 from typing import List
+from typing import Tuple
 
 from intrinsic_camera_calibrator.board_detections.board_detection import BoardDetection
 from intrinsic_camera_calibrator.calibrators.calibrator import Calibrator
-from intrinsic_camera_calibrator.camera_model import CameraModel
+from intrinsic_camera_calibrator.camera_models.camera_model import CameraModelEnum
+from intrinsic_camera_calibrator.camera_models.camera_model_factory import make_camera_model
+from intrinsic_camera_calibrator.camera_models.ceres_camera_model import CeresCameraModel
 from intrinsic_camera_calibrator.parameter import Parameter
 
 
 class CeresCalibrator(Calibrator):
     def __init__(self, lock: threading.RLock, cfg: Dict = {}):
         super().__init__(lock, cfg)
+        self.radial_distortion_coefficients = Parameter(int, value=3, min_value=0, max_value=3)
+        self.rational_distortion_coefficients = Parameter(int, value=3, min_value=0, max_value=3)
+        self.use_tangential_distortion = Parameter(
+            bool, value=True, min_value=False, max_value=True
+        )
+        self.pre_calibration_num_samples = Parameter(int, value=6, min_value=0, max_value=20)
```

Comment:
This number should be as high as possible taking into consideration that opencv does not scale well in terms of computational complexity (e.g., we limit it to 80 samples by default. when sergio enables a few decomposition options it will increase to a few hundred)

Reply:
Sorry for being insistent, but can you leave it as a higher number? (in the experiments I remember doing with over 40 is was way more stable. How was it for you?)

Reply:
Confirmed, addressed in https://github.com/tier4/CalibrationTools/pull/208/commits/e5a1de6b5896d904da39babd9618356680951e8e